libraries/AP_Motors/AP_MotorsHeli.h

```diff
@@ -203,6 +227,8 @@ class AP_MotorsHeli : public AP_Motors {
         uint8_t in_autorotation         : 1;    // true if aircraft is in autorotation
         uint8_t enable_bailout          : 1;    // true if allowing RSC to quickly ramp up engine
         uint8_t servo_test_running      : 1;    // true if servo_test is running
+        uint8_t land_complete           : 1;    // true if aircraft is landed
```

Comment:
this makes heliflags 16 bits, need to check that is OK (is it logged somewhere as 8 bit?)

Reply:
From what I can tell, only accessors to individual flags are used.